admin/database/postgres/postgres.go

```diff
@@ -696,6 +696,11 @@ func (c *connection) DeleteProjectMemberUser(ctx context.Context, projectID, use
 	return checkDeleteRow("project member", res, err)
 }
 
+func (c *connection) DeleteAllProjectMemberUserForOrg(ctx context.Context, orgID, userID string) error {
```

Comment:
nit: `DeleteAllProjectMemberUserForOrg` -> `DeleteAllProjectMemberUserForOrganization` for consistency